NiftyPerforce/P4Operations.cs

```diff
@@ -63,6 +63,16 @@ private static void UnlockOp(bool ok, object token_)
             }
         }
 
+        private static string EscapeP4Path(string filename)
+        {
+            return filename.Replace("%", "%25").Replace("#", "%23").Replace("@", "%40");

```

Comment:
Do you think for completeness we should replace "*" with "%2A"?

Reply:
Totally with you that having * inside file name can be a nightmare. Good news is that Windows does not allow "*" in file/folder name. Thus I think we don't have to worry about that for now 😜

Reply:
Hehe fair enough :)